src/pages/Accounts.vue

```diff
@@ -75,37 +91,60 @@
 <script>
 import { mapState, mapGetters, mapMutations } from 'vuex';
 import {
-  AeIdentity, AeIdentityAvatar, AeButton, AeDivider,
-  AeLabel, AeInput, AeModalLight,
-} from '@aeternity/aepp-components';
+  AeButton, AeLabel, AeInput, AeModalLight,
+  AeAddress, AeList, AeListItem,
+} from '@aeternity/aepp-components-3';
+import { swiper as Swiper, swiperSlide as SwiperSlide } from 'vue-awesome-swiper';
 import MobilePage from '../components/MobilePage.vue';
+import AeAccount from '../components/AeAccount.vue';
 import FixedAddButton from '../components/FixedAddButton.vue';
-import ListItem from '../components/ListItem.vue';
-import AeRadio from '../components/AeRadio.vue';
-import { roundToken } from '../lib/filters';
+import Guide from '../components/Guide.vue';
 
 export default {
   components: {
-    AeIdentity,
-    AeIdentityAvatar,
+    AeAccount,
+    AeAddress,
     AeButton,
-    AeDivider,
     MobilePage,
     FixedAddButton,
     AeLabel,
     AeInput,
     AeModalLight,
-    ListItem,
-    AeRadio,
+    Swiper,
+    SwiperSlide,
+    Guide,
+    AeList,
+    AeListItem,
   },
-  filters: { roundToken },
   data: () => ({
     modalVisible: false,
     newAccountName: '',
   }),
   computed: {
-    ...mapGetters(['totalBalance', 'activeIdentity', 'identities']),
-    ...mapState(['selectedIdentityIdx']),
+    ...mapGetters(['activeIdentity', 'identities']),
+    ...mapState({
+      selectedIdentityIdx: ({ selectedIdentityIdx }) => selectedIdentityIdx,
+    }),
+    swiperOptions() {
+      const account = this;
+
+      function switchAccount() {
+        account.selectIdentity(this.activeIndex);
+      }
+
+      return {
+        spaceBetween: 10,
+        centeredSlides: true,
+        pagination: {
+          el: '.swiper-pagination',
+          clickable: true,
+        },
+        initialSlide: this.selectedIdentityIdx,
+        on: {
+          slideChange: switchAccount,
```

Comment:
```suggestion
          slideChange() {
            self.selectIdentity(this.activeIndex);
          },
```
and also remove `switchAccount` and rename `account` to `self`. Swiper has so inconvenient API 😕